libraries/common/components/InfiniteContainer/index.jsx

```diff
@@ -86,6 +88,10 @@ class InfiniteContainer extends Component {
    * @param {Object} nextProps The next props.
    */
   componentWillReceiveProps(nextProps) {
+    if (nextProps.sortOrder !== this.props.sortOrder) {
```

Comment:
This component should not know about request specifics. Currently it won't for example execute `resetComponent` when filters changed (by removing it by using chips).

Instead sortOrder, we might want to pass a "requestHash" or something like that. Then the component which uses InfiniteContainer would have to produce a requestHash, which then would be created with those specifics in mind.

When this changed, InfiniteContainer would reset.

Reply:
good point. changed it